sites/docs/pages/components/bar-chart.md

```diff
@@ -8,6 +8,8 @@ queries:
 - categories_by_channel.sql
 ---
 
+<BarChart data={[]} emptySet="pass" />
```

Comment:
This is presumeably just for testing - should be deleted

Reply:
[852f0a9](https://github.com/evidence-dev/evidence/pull/2628/commits/852f0a93967e126ade2c08fa48e83b1538263e6e)